Add Information About the include Folder to Develop Your Astro Project (#1504)
astronomer/docs: astro/develop-project.md

To rebuild your project after making a change to any of these files, you must [restart your local environment](develop-project.md#restart-your-local-environment).

> ### Deploy only DAGs

Reviewer: ```suggestion
## Add utility files for DAGs
```
The purpose of the parent topic is more to discuss whether the user needs to restart their project to deploy any local changes. I think this should be its own H2 topic closer to where we discuss other folders, like near "Add Python and OS-level packages"

> With Astro CLI 1.7 and later, you can run `astro deploy -—dags` to push only the `dags` directory of your Astro project to a Deployment on Astro. This is an additional option to `astro deploy` that makes for a faster development experience and allows you more flexibility when configuring your CI/CD processes.

Reviewer: ```suggestion
Use the `include` folder to store any additional utilities your DAGs need, such as helper functions, templated SQL scripts, and custom operators.
```

Reviewer: This topic seems a bit duplicative to our existing DAG deploy documentation. I tried making it primarily about the `include` folder, with a note about how this works with DAG deploys.

> When you use the `astro deploy -—dags` command, the `include` folder in the Astro project directory is not deployed with the DAG and is instead included with the image by default. If you are deploying only DAGs, Astronomer recommends that you create a new folder in the `dags` directory of your local project to store Python functions or other files that are required by the DAG.
> 
> To use `astro deploy -—dags`, you need to enable it. See [Enable DAG-only deploys](deploy-code.md#enable-dag-only-deploys).

Reviewer: ```suggestion

:::tip

When you use the `astro deploy -—dags` command, the `include` folder in the Astro project directory is not deployed with your DAGs and is instead build into the Docker image with your other project files.

If you are deploying only DAGs, Astronomer recommends moving the `include` folder into the `dags` directory so that your DAGs can quickly access your utility files. See [Deploy DAGs only](deploy-code.md#deploy-dags-only).

:::
```

Tried to make the benefit of moving the `include` folder more clear here, and also don't think we need to discuss enabling the DAG deploy feature in this topic. It seems better to guide users to the topic in general and give them the option to read the enabling topic if they want to proceed with DAG deploys.

To rebuild your project after making a change to any of these files, you must [restart your local environment](develop-project.md#restart-your-local-environment).

> ## Add utility files for DAGs

Author: ```suggestion
## Add DAG utility files
```

Use the `include` folder to store additional utilities required by your DAGs. For example, helper functions, templated SQL scripts, and custom operators.

:::tip

> When you use the `astro deploy -—dags` command, the `include` folder in the Astro project directory is not deployed with your DAGs and is instead build into the Docker image with your other project files.

Author: ```suggestion
When you use the `astro deploy -—dags` command, the `include` folder in the Astro project directory is not deployed with your DAGs and is instead built into the Docker image with your other project files.
```

> If you are deploying only DAGs, Astronomer recommends moving the `include` folder into the `dags` directory so that your DAGs can quickly access your utility files. See [Deploy DAGs only](deploy-code.md#deploy-dags-only).

Author: ```suggestion
If you are deploying only DAGs, Astronomer recommends moving the `include` folder into the `dags` directory so that your DAGs can access your utility files. See [Deploy DAGs only](deploy-code.md#deploy-dags-only).
```
